packages/react-charting/docs/TestPlans/StackedBarChart/ComponentTests.md

```diff
@@ -0,0 +1,58 @@
+|                      - HideTooltip prop set to “false”                       |                               Should mount callout when hideTootip is false                               |    Enzyme     |
+|                       - HideDenominator set to “false”                       |                                         Should not mount callout                                          |    Enzyme     |
+|                    - onRenderCalloutPerDataPoint is given                    |                                 Should render onRenderCalloutPerDataPoint                                 |    Enzyme     |
+|                  - onRenderCalloutPerDataPoint is not given                  |                               Should not render onRenderCalloutPerDataPoint                               |    Enzyme     |
+|               Test 3: Render calling with respective to props                |                                                                                                           |               |
+| - No prop changes: Mount stacked bar chart and then set the same props again |                               Render function should have been called twice                               |    Enzyme     |
+|   - Prop changes: Mount stacked bar chart and then set the some other prop   |                               Render function should have been called twice                               |    Enzyme     |
+|                             Test 4: Mouse events                             |                                                                                                           |               |
+|                            - Mouse over on a bar                             |                               Should render callout correctly on mouseover                                |    Enzyme     |
+|                - Mouse over on a bar with customized callout                 |                               Should render customized callout on mouseover                               |    Enzyme     |
+|                            - Mouse move on a bar                             |                               Should render callout correctly on mousemove                                |    Enzyme     |
+|        Test 5: Render empty chart aria label div when chart is empty         |                                                                                                           |               |
+|               - Stacked bar chart mounted with non-empty data                |                                  No empty chart aria label div rendered                                   |    Enzyme     |
+|                 - Stacked bar chart mounted with empty data                  |                                    Empty chart aria label div rendered                                    |    Enzyme     |
```

Comment:
I think stacked bar chart with empty data shows just the benchmark. Or is it applicable for chart with data value = 0?

Reply:
With empty data, it does not show anything, but with zero data, it shows empty transparent bars. If we specify the benchmark prop, it shows the benchmark on top of it.